docs/hello_nextflow/02_hello_world.md

```diff
@@ -822,13 +833,13 @@ In most cases, when we run on multiple inputs, the input values are contained in
 
 Nextflow offers a variety of predefined operators and functions for reading data in from common file formats and applying text transformations. In this example, we used the [`fromPath()`](https://www.nextflow.io/docs/latest/channel.html#frompath) channel factory with the [`splitText()`](https://www.nextflow.io/docs/latest/operator.html#splittext) operator to read each line as a separate value, then we used a [closure](https://www.nextflow.io/docs/latest/script.html) to apply the [`trim()`](https://www.tutorialspoint.com/java/java_string_trim.htm) method to strip the newline (`\n`) character from each element.
 
-#### 1. Modify the channel declaration to take an input file (through a parameter) instead of hardcoded values
+#### 1. Modify the channel declaration to take an input file (through a parameter) instead of a single parameter
 
 _Before:_
 
 ```groovy title="hello-world.nf" linenums="38"
 // create a channel for inputs
-greeting_ch = Channel.of('Hello','Bonjour','Holà')
+greeting_ch = Channel.of(params.greeting)
```

Comment:
This is incorrect-- remember this is the 'before' block, and we changed the input back to hardcoded values in the previous section